tests/rules/test_key_ordering.py

```diff
@@ -103,14 +103,38 @@ def test_accents(self):
                    'haïr: true\n'
                    'hais: true\n', conf,
                    problem=(3, 1))
-        self.check('---\n'
-                   'haïr: true\n'
-                   'hais: true\n', conf,
-                   problem=(3, 1))
```

Comment:
Could you keep this one? It makes sure an error is reported when `locale` isn't provided.

Reply:
Those 4 lines were identically repeated. So that test is still in there, just once now and not twice. Should have left a comment here, sorry ;)